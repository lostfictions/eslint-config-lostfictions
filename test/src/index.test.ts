import { expect, describe, it } from "vitest";

// eslint-disable-next-line vitest/valid-title
describe("", () => {});

describe("my cool test", () => {
  it("doesn't require hints for a test with a single snapshot", () => {
    expect(2).toMatchSnapshot();
  });

  it("requires hints for a test with multiple snapshots", () => {
    expect(1).toMatchSnapshot("a cool hint");
    // eslint-disable-next-line vitest/prefer-snapshot-hint
    expect(2).toMatchSnapshot();
  });
});
