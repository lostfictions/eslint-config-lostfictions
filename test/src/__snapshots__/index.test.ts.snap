// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`my cool test > doesn't require hints for a test with a single snapshot 1`] = `2`;

exports[`my cool test > requires hints for a test with multiple snapshots > a cool hint 1`] = `1`;

exports[`my cool test > requires hints for a test with multiple snapshots 1`] = `2`;
